src/index.css

```diff
@@ -83,15 +86,297 @@ input:active {
   background-color: #dfdfdf;
 }
 
-.buttons button {
+button .buttons {
   border: 1px solid #d0d0d0;
 }
 
-.buttons button:nth-child(17) {
+button:nth-child(17) .buttons {
   grid-area: 5/1/5/3;
 }
 
+button {
+  font-size: 15px;
+  margin: 10px auto;
+  padding: 5px;
+  width: 100px;
+  height: 35px;
+  background-color: white;
+  border: 2px solid black;
+}
+
 .buttons button:nth-child(4n),
 .buttons button:last-child {
   background-color: #ff8e00;
 }
+
+nav {
+  width: 100%;
+}
+
+.website-top {
+  text-align: center;
+  border: 5px solid black;
+  margin-bottom: 10px;
+  background: lightgrey;
+}
```

Comment:
- [ ] [OPTIONAL] Great job following the layout 👍 Please note that this part of the wireframe doesn't need to be included in your design 🙏 

![image](https://user-images.githubusercontent.com/74814780/184101061-df44e489-d3c1-4a90-b5e4-69466f878c53.png)